provisioning_client/tests/prov_x509_client_e2e/prov_x509_client_e2e.c

```diff
@@ -44,14 +48,31 @@ BEGIN_TEST_SUITE(prov_x509_client_e2e)
         g_dps_scope_id = getenv(DPS_ID_SCOPE);
         ASSERT_IS_NOT_NULL(g_dps_scope_id, "DPS_ID_SCOPE is NULL");
 
+#ifdef HSM_TYPE_X509
+        char* dps_x509_cert_individual_base64 = getenv(DPS_X509_INDIVIDUAL_CERT_BASE64);
+        ASSERT_IS_NOT_NULL(dps_x509_cert_individual_base64, "DPS_X509_INDIVIDUAL_CERT_BASE64 is NULL");
```

Comment:
nit: here and below in the logs
```suggestion
        ASSERT_IS_NOT_NULL(dps_x509_cert_individual_base64, "env var DPS_X509_INDIVIDUAL_CERT_BASE64 is not set");
```
just to notify the user of the specifics of the problem.